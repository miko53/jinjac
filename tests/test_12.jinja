Essai title
{{ "test title second one" | title() }}
{{ "TEST TITLE SECOND ONE" | title() }}
Fin de test 

