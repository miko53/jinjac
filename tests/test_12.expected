Essai title
Test Title Second One
Test Title Second One
Fin de test 

